docs/configuration/theme.md

```diff
@@ -31,3 +31,5 @@ There are currently 3 available themes for Authelia:
 * light (default)
 * dark
 * grey
+
+To enable automatic switching between themes, you can set `theme` to `auto`. The theme will be set to either `dark` or `light` depending on the user's system preference.
```

Comment:
```suggestion
To enable automatic switching between themes, you can set `theme` to `auto`. The theme will be set to either `dark` or `light` depending on the user's system preference which is determined using media queries. To read more technical details about the media queries used, read the [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme).
```

Reply:
SGTM.

Reply:
SGTM to me too. That's what I had in mind.

Reply:
Fixed.